src/modules/extendedCallWindow/i18n/en_US.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
             [/^Large fire boat(s)?$/]: [24],
             [/^Large rescue boat(s)?$/]: [25],
             [/^Light Boat(s)?$/]: [25],
-            [/^Wildland fire engine(s)?$/]: [30, 31, 32, 38, 39, 40],
+            [/^Wildland fire vehicle(s)?$/]: [30, 31, 32, 38, 39, 40],
```

Comment:
Can I just ask why this got changed?

Reply:
because ingame name is "vehicles" and not "engines"